Makefile

```diff
@@ -34,7 +34,7 @@ all: Makefile .docker-image conf
 
 .PHONY: bash
 bash:
-	$(call docker_run, /bin/bash)
+	$(call docker_run, sh -c "${SOURCE_OE}; exec /bin/bash")
```

Comment:
where is SOURCE_OE defined?

Reply:
That's missing from the commit. I'll add it back.